.gitignore

```diff
@@ -137,9 +137,9 @@ publish/
 # NuGet Packages
 *.nupkg
 # The packages folder can be ignored because of Package Restore
-**/packages/*
+#**/packages/*
```

Comment:
Consider renaming the packages folder to something more descriptive so we we can maintain this ignoring **/packages/*